fastlane_bot/events/event_gatherer.py

```diff
@@ -86,7 +86,6 @@ async def _get_logs_recursive(self, from_block: int, to_block: int, topics: List
                     "topics": topics
                 })
             except Exception as e:
-                assert "eth_getLogs" in str(e), str(e)
```

Comment:
Please log any error-message which **doesn't** contain the `"eth_getLogs"` part in it.

This error is well expected as part of this binary-search, but any other error is something that we would want to be aware of.

For example, network disconnection, or even some silly coding mistake inside the `try` part.

Otherwise, we're essentially masking out potential bugs and issues.